.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: Test
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  test_lint:
+    name: Linting tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup python
+        uses: actions/setup-python@v4.5.0
+        with:
+          python-version: "3.11.1"
+
+      - name: Setup linting
+        run: pip install black==22.8.0
```

Comment:
Enjoying the new test workflow, can be a model going forward for our other data products. Can add this to the template repo